packages/connection-router-api/src/msg.rs

```diff
@@ -0,0 +1,47 @@
+use crate::primitives::*;
+use cosmwasm_schema::{cw_serde, QueryResponses};
+
+#[cw_serde]
+pub enum ExecuteMsg {
+    /*
+     * Governance Methods
+     * All of the below messages can only be called by governance
```

Comment:
```suggestion
     * All of the below messages should only be called by governance
```
Since this is just an API, we can't enforce what the impl does